src/SmartComponents/Inventory/applications/vulnerabilities/DataMapper.js

```diff
@@ -18,13 +10,21 @@ export function createCveListBySystem({ isLoading, ...rest }) {
             data: data.map(row => ({
                 id: row.id,
                 cells: [
-                    <span key={ row.id }>
-                        <SecurityIcon size="md" color={ colorToImpact[row.attributes.impact] } />
-                    </span>,
+                    <Shield
+                        impact={ row.attributes.impact }
+                        hasTooltip={ true }
+                        tooltipPosition={ 'right' }
```

Comment:
Tooltip needs to be on the top

Reply:
I have an assignment to be right because it is the first column in the table. When the window size width is small, the `top` position of tooltip may not be displayed correctly:
![screenshot 2019-02-28 08 31 29](https://user-images.githubusercontent.com/1569401/53549068-89a25980-3b33-11e9-9feb-8ad6dedc2b6e.png)

Should I use `top` position anyway?

Reply:
🤔 This may be a problem with the tooltip component from the PF side. Let me investigate further. Make a note and we'll merge it for now.